packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/templates/dynamicNavigation.html

```diff
@@ -5,9 +5,16 @@
   </template>
   <template lwc:else>
     <template if:false={hideSort}>
-      <div class="slds-var-m-bottom_small slds-clearfix">
+      <template lwc:if={shouldDisplaySortByTitle}>
+        <div class="slds-align_absolute-center filters-header slds-var-m-bottom_small">
+          <div data-cy="filters-title" class="slds-text-heading_small">
+            {labels.sortBy}
+          </div>
+        </div>
+      </template>
```

Comment:
This should not be needed, the label should be included in the sort component.

Reply:
Additionally how could this be less confusing?
Cause here we will end up with a component that with a certain variant displays a label and with another it doesn't. and the variants does not refer to labels at all. for me that's a confusion.

Cause it's not like our variants are `with-label` and `label-free` our variants are size oriented.